Upgrade fabric (#260)
citusdata/test-automation: README.md

If your subscriptions list doesn't contain `Azure SQL DB Project Orcas - CitusData`, to add it, contact someone who is authorized.

> 4. You should use `ssh-agent` to add your ssh keys, which will be used for downloading release-test-results repository. Note that your keys are kept only in memory, therefore this is a secure step.

Reviewer: ```suggestion
4. You should use `ssh-agent` to add your ssh keys, which will be used to upload to the release-test-results repository. Note that your keys are kept only in memory, therefore this is a secure step.
```

    ```bash
    # start ssh agent
    eval `ssh-agent -s`

>     # Add your Github ssh key to install release-test-results repo

Reviewer: ```suggestion
    # Add your Github ssh key to upload results to release-test-results repo
```
